### DISCUSSION TITLE
wdi5 vs iRPA

### DISCUSSION THREAD

**Discussion by marco-paciucci** [ui5-community/wdi5#328] [Q&A]
Hello,
I don't know if this is the right section but I would need to get a better understanding as to what is the recommended method for E2E testing Fiori Apps delivered on SAP Cloud Foundry Fiori Launchpad.
I know how to use wdi5 but I think that there's another viable option which is iRPA.
Can anybody shed some lights along with pros and cons of each solution?
Thanks

**Comment by vobu** [comment]
I took the liberty and moved the original issue to a discussion, as this is the proper place for your question
